pgmq-extension/README.md

```diff
@@ -9,6 +9,8 @@ A lightweight message queue. Like [AWS SQS](https://aws.amazon.com/sqs/) and [RS
 
 **Source**: https://github.com/pgmq/pgmq
 
+<img src="https://github.com/user-attachments/assets/3e6d23c5-83fa-4c4a-98ae-77f305e3cd5c" width="75%" alt="pgmq demo gif">
```

Comment:
I suggest using Markdown:

```suggestion
![pgmq demo gif"](https://github.com/user-attachments/assets/3e6d23c5-83fa-4c4a-98ae-77f305e3cd5c)
```

Reply:
When I looked at the source on the render I didn't see the 75% preserved, so assumed it wouldn't matter. I don't think GFM supports attributes on images. [Details here](https://github.github.com/gfm/#images).

Reply:
the width seems to work? I just shrunk it down to 10% as an example. This is what I see now:

<img width="556" alt="image" src="https://github.com/user-attachments/assets/6d37334d-7628-4097-b589-fb46d36d96e5" />

https://github.com/pgmq/pgmq/tree/demo-gif?tab=readme-ov-file#postgres-message-queue-pgmq

Reply:
LOL, okay, go for it.

Reply:
I'll change it to markdown soon as I make more appropriately sized content